components/NewsroomYoutube.js

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState, useRef } from 'react';
+import TextTruncate from 'react-text-truncate'
+import ArrowLeft from './icons/ArrowLeft';
+import ArrowRight from './icons/ArrowRight';
+import TextLink from './typography/TextLink';
+import tempData from '../config/videos.json';
+import Heading from './typography/Heading'
+import Paragraph from './typography/Paragraph'
+
+export default function NewsroomYoutube({ className = '' }) {
+  const [videoData, setvideoData] = useState([]);
+  const videoContainer = useRef();
+  const videoref = useRef();
+  const shiftLeft = (e) => {
+    e.preventDefault();
+    const width = videoref.current.clientWidth;
+    videoContainer.current.scrollLeft -= width + 16;
+  };
+  const shiftRight = (e) => {
+    e.preventDefault();
+    const width = videoref.current.clientWidth;
+    videoContainer.current.scrollLeft += width + 16;
+  };
+
+  let buttonClass = 'bg-secondary-100 hover:bg-secondary-500 shadow-md rounded border-secondary-500 border text-secondary-500 hover:text-white focus:outline-none mx-2';
+  useEffect(() => {
+    // async function youtube() {
```

Comment:
The API call is restricted here because we have per day limited quota of the Youtube API calls. Hence, it will be uncommented in the last commit. The API call function has been tested properly and is working fine.